src/canvas-extractor/canvas_extractor/__main__.py

```diff
@@ -145,11 +137,8 @@ def _get_assignments(
     (assignments, udm_assignments_df) = extract_assignments(
         courses, sections_df, sync_db
     )
-    write_multi_csv(
-        udm_assignments_df,
-        datetime.now(),
-        os.path.join(output_directory, ASSIGNMENT_ROOT_DIRECTORY),
-    )
+    logger.info("Writing LMS UDM Assignments to CSV files")
+    write_assignments(udm_assignments_df, datetime.now(), output_directory)
```

Comment:
Would there be any value to calculating the datetime once and passing the same file to each `write_xyz` statement?